### PR TITLE
Improve styling and organization of element panel

### PR DESCRIPTION
**Pull request by pbakaus**
Follow up to #639 (please merge that other one first). This gets the element panel much closer to desired styling:

image

Outstanding thing: The "Add text" button in the Text Panel needs to be aligned - technically easy, but I'm not very familiar yet with Styled Components and how to extend/nest styles of a existing component (maybe @wassgha or @swissspidy can do a quick follow up here).

### CODE REVIEW

**Comment by pbakaus** [comment]
(this looks big because it contains the changes from #639)

File: assets/src/edit-story/components/library/panes/media/mediaPane.js [hunk 1]
```diff
@@ -221,33 +224,29 @@ function MediaPane(props) {
...
-              >
-                {name}
-              </FilterButton>
-            ))}
-          </FilterButtons>
+          <FilterArea>
+            <FilterButtons>
+              {FILTERS.map(({ filter, name }, index) => (
+                <FilterButton
+                  key={index}
```
Comment (carloskelly13) [inline assets/src/edit-story/components/library/panes/media/mediaPane.js:238]: I’m guessing there isn’t anything more unique than the array index to use as the key?

**Review by carloskelly13 — APPROVED** [review]
One minor comment, but looks good. I did also notice, not part of this diff, but the `MediaPane` component had some callback functions not wrapped in `useCallback`. Unsure of the perf hit, but may be worth memoizing.

File: assets/src/edit-story/components/library/panes/media/mediaPane.js (hunk 1 above)
Comment (pbakaus) [reply assets/src/edit-story/components/library/panes/media/mediaPane.js:238]: good nit, but I only reshuffled this code, didn't add it.. not sure who did originally, maybe @spacedmonkey knows

Comment (merapi) [reply assets/src/edit-story/components/library/panes/media/mediaPane.js:238]: Not my code but: These elements never change their position, so it's perfectly safe to use `index`, CMIIW.

**Pull request merged**